apps/stories/api_views.py

```diff
@@ -247,11 +247,24 @@ def post(self, request, format=None):
 
 class StoryInfoView(KLPAPIView):
     def get(self, request):
+        school_type = BoundaryType.objects.get(name='Primary School')
+        school_qset = School.objects.filter(
+            admin3__type=school_type, status=2, id__gte=settings.STATE_STARTING_SCHOOL_ID
+        ).order_by().values('id')
+       
+        stories_qset = Story.objects.select_related(
+            'school', 'user'
+        ).filter(
+            school__in=school_qset
+        )
```

Comment:
Obviously this can be implemented much nicer, but the above snippet us getting all the schools for Odisha and filtering the `Story` with those schools. All `Story` querysets will be empty to begin with.